custom/icds_reports/tasks.py

```diff
@@ -728,7 +716,7 @@ def _find_stagnant_cases(adapter):
     query = query.with_entities(table.columns.doc_id).filter(
         table.columns.inserted_at <= stagnant_date
     ).distinct()
-    return query.all()
+    return set(query.all())
```

Comment:
a smart man would probably paginate this query as well, but that can be a followup as i think this is strictly better